sympy/functions/special/bsplines.py

```diff
@@ -137,6 +137,20 @@ def bspline_basis(d, knots, n, x):
         >>> f = lambdify(x, b0)
         >>> y = f(0.5)
 
+    Parameters
```

Comment:
I'm not sure about this, but I think the Parameters would be better before examples.
It might be better to show the users what each parameter means, before showing them examples of how to use the function.

Reply:
Even I think so. But from the beginning in all docs the parameters section has been added after Examples section. Should I change the whole thing?